### ISSUE TITLE
naked start

### ISSUE THREAD

**Issue by arxdsilva** [prest/prest#767]
version 1.2.0
```go
prest-prest-1     | 2023/01/18 20:39:35 [error] pq: no pg_hba.conf entry for host "192.168.192.3", user "postgres", database "prest", no encryption
```

**Comment by vmesel** [comment]
@arxdsilva what should be the expected default behaviour?

**Comment by arxdsilva** [comment]
Should be able to start without errors

**Comment by vmesel** [comment]
@arxdsilva just tried out to download the repo and `docker-compose up`:

```
λ ~/Github/prest/ main git status
On branch main
Your branch is up to date with 'origin/main'.

nothing to commit, working tree clean
λ ~/Github/prest/ main
λ ~/Github/prest/ main docker-compose up
[+] Running 1/0
 ⠿ Container prest-postgres-1  Created                                                                         0.0s
Attaching to prest-postgres-1, prest-prest-1
prest-postgres-1  | The files belonging to this database system will be owned by user "postgres".
prest-postgres-1  | This user must also own the server process.
prest-postgres-1  |
prest-postgres-1  | The database cluster will be initialized with locale "en_US.utf8".
prest-postgres-1  | The default database encoding has accordingly been set to "UTF8".
prest-postgres-1  | The default text search configuration will be set to "english".
prest-postgres-1  |
prest-postgres-1  | Data page checksums are disabled.
prest-postgres-1  |
prest-postgres-1  | fixing permissions on existing directory /var/lib/postgresql/data ... ok
prest-postgres-1  | creating subdirectories ... ok
prest-postgres-1  | selecting dynamic shared memory implementation ... posix
prest-postgres-1  | selecting default max_connections ... 100
prest-postgres-1  | selecting default shared_buffers ... 128MB
prest-postgres-1  | selecting default time zone ... Etc/UTC
prest-postgres-1  | creating configuration files ... ok
prest-postgres-1  | running bootstrap script ... ok
prest-postgres-1  | performing post-bootstrap initialization ... ok
prest-postgres-1  | syncing data to disk ... ok
prest-postgres-1  |
prest-postgres-1  |
prest-postgres-1  | Success. You can now start the database server using:
prest-postgres-1  |
prest-postgres-1  |     pg_ctl -D /var/lib/postgresql/data -l logfile start
prest-postgres-1  |
prest-postgres-1  | initdb: warning: enabling "trust" authentication for local connections
prest-postgres-1  | initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
prest-postgres-1  | waiting for server to start....2023-01-26 15:56:04.451 UTC [56] LOG:  starting PostgreSQL 15.1 (Debian 15.1-1.pgdg110+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 64-bit
prest-postgres-1  | 2023-01-26 15:56:04.454 UTC [56] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
prest-postgres-1  | 2023-01-26 15:56:04.498 UTC [59] LOG:  database system was shut down at 2023-01-26 15:55:57 UTC
prest-postgres-1  | 2023-01-26 15:56:04.591 UTC [56] LOG:  database system is ready to accept connections
prest-postgres-1  |  done
prest-postgres-1  | server started
prest-postgres-1  | CREATE DATABASE
prest-postgres-1  |
prest-postgres-1  |
prest-postgres-1  | /usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*
prest-postgres-1  |
prest-postgres-1  | 2023-01-26 15:56:16.016 UTC [56] LOG:  received fast shutdown request
prest-postgres-1  | waiting for server to shut down....2023-01-26 15:56:16.021 UTC [56] LOG:  aborting any active transactions
prest-postgres-1  | 2023-01-26 15:56:16.026 UTC [56] LOG:  background worker "logical replication launcher" (PID 62) exited with exit code 1
prest-postgres-1  | 2023-01-26 15:56:16.026 UTC [57] LOG:  shutting down
prest-postgres-1  | 2023-01-26 15:56:16.030 UTC [57] LOG:  checkpoint starting: shutdown immediate
prest-postgres-1  | .2023-01-26 15:56:17.934 UTC [76] FATAL:  the database system is shutting down
prest-postgres-1  | .2023-01-26 15:56:18.393 UTC [57] LOG:  checkpoint complete: wrote 918 buffers (5.6%); 0 WAL file(s) added, 0 removed, 0 recycled; write=2.320 s, sync=0.004 s, total=2.368 s; sync files=250, longest=0.002 s, average=0.001 s; distance=4217 kB, estimate=4217 kB
prest-postgres-1  | 2023-01-26 15:56:18.511 UTC [56] LOG:  database system is shut down
prest-postgres-1  |  done
prest-postgres-1  | server stopped
prest-postgres-1  |
prest-postgres-1  | PostgreSQL init process complete; ready for start up.
prest-postgres-1  |
prest-postgres-1  | 2023-01-26 15:56:18.722 UTC [1] LOG:  starting PostgreSQL 15.1 (Debian 15.1-1.pgdg110+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 64-bit
prest-postgres-1  | 2023-01-26 15:56:18.723 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
prest-postgres-1  | 2023-01-26 15:56:18.723 UTC [1] LOG:  listening on IPv6 address "::", port 5432
prest-postgres-1  | 2023-01-26 15:56:18.734 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
prest-postgres-1  | 2023-01-26 15:56:18.911 UTC [80] LOG:  database system was shut down at 2023-01-26 15:56:18 UTC
prest-postgres-1  | 2023-01-26 15:56:19.049 UTC [1] LOG:  database system is ready to accept connections
prest-postgres-1  | 2023-01-26 15:56:48.113 UTC [91] FATAL:  role "root" does not exist
prest-prest-1     | [prestd] Waiting for port postgres: to become available...
prest-prest-1     | [prestd] Plugin/build: starting...
prest-prest-1     | [prestd] Go build simple plugins in only file!
prest-prest-1     | go build: hello plugin...
prest-prest-1     | [prestd] Go build complex plugins in folder (with main.go file)!
prest-prest-1     | [prestd] Plugin/build: ending
prest-prest-1     | [prestd] Ready hosting postgres to port  !
prest-prest-1     | 2023/01/26 15:57:16 [warning] file './prest.toml' not found, falling back to default settings
prest-prest-1     | 2023/01/26 15:57:16 [warning] read env config error: Config File "prest" Not Found in "[/app]"
prest-prest-1     | 2023/01/26 15:57:16 [warning]
prest-prest-1     | You are using v1 of prestd configs, please migrate to v2.
prest-prest-1     |
prest-prest-1     | v1 will be deprecated soon.
prest-prest-1     |
prest-prest-1     | View more at https://docs.prestd.com/prestd/deployment/server-configuration
prest-prest-1     | 2023/01/26 15:57:16 [warning] adapter is not set. Using the default (postgres)
prest-prest-1     | 2023/01/26 15:57:17 [warning] You are running prestd in public mode.
prest-prest-1     | 2023/01/26 15:57:17 [warning] command.go:920 You are running prestd in debug mode.
prest-prest-1     | [prestd] listening on 0.0.0.0:3000 and serving on /
prest-postgres-1  | 2023-01-26 15:57:18.400 UTC [102] FATAL:  role "root" does not exist
```

I'm not able to replicate `pg_hba.conf` error.

**Comment by arxdsilva** [comment]
fixed with #775's fix
